.github/workflows/wheel.yml

```diff
@@ -0,0 +1,416 @@
+            manylinux: manylinux_2_28
+            python: cp38-manylinux_x86_64
+            arch: x86_64
+          - build: CPython 3.9 64 bits manylinux_2_28
+            manylinux: manylinux_2_28
+            python: cp39-manylinux_x86_64
+            arch: x86_64
+          - build: CPython 3.10 64 bits manylinux_2_28
+            manylinux: manylinux_2_28
+            python: cp310-manylinux_x86_64
+            arch: x86_64
+          - build: CPython 3.11 64 bits manylinux_2_28
+            manylinux: manylinux_2_28
+            python: cp311-manylinux_x86_64
+            arch: x86_64
+          - build: CPython 3.12 64 bits manylinux_2_28
+            manylinux: manylinux_2_28
+            python: cp312-manylinux_x86_64
+            arch: x86_64
+          - build: CPython 3.13 64 bits manylinux_2_28
+            manylinux: manylinux_2_28
+            python: cp313-manylinux_x86_64
+            arch: x86_64
+          # -------------------------------------------------------------------
+          # CPython 64 bits manylinux2014
+          # -------------------------------------------------------------------
+          - build: CPython 3.8 64 bits manylinux2014
+            manylinux: manylinux2014
+            python: cp38-manylinux_x86_64
+            arch: x86_64
+          - build: CPython 3.9 64 bits manylinux2014
+            manylinux: manylinux2014
+            python: cp39-manylinux_x86_64
+            arch: x86_64
+          - build: CPython 3.10 64 bits manylinux2014
+            manylinux: manylinux2014
+            python: cp310-manylinux_x86_64
+            arch: x86_64
+          - build: CPython 3.11 64 bits manylinux2014
+            manylinux: manylinux2014
+            python: cp311-manylinux_x86_64
+            arch: x86_64
+          - build: CPython 3.12 64 bits manylinux2014
+            manylinux: manylinux2014
+            python: cp312-manylinux_x86_64
+            arch: x86_64
+          - build: CPython 3.13 64 bits manylinux2014
+            manylinux: manylinux2014
+            python: cp313-manylinux_x86_64
+            arch: x86_64
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        name: Install Python
+        with:
+          python-version: '3.9'
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+        with:
+          platforms: all
+
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.21.1
+        env:
+          CIBW_BUILD: ${{ matrix.python }}
+          CIBW_ARCHS: ${{ matrix.arch }}
+          CIBW_MANYLINUX_X86_64_IMAGE: ${{ matrix.manylinux }}
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: cibw-wheels-${{ matrix.python }}-${{ matrix.manylinux }}
+          path: ./wheelhouse/*.whl
+
+  # ---------------------------------------------------------------------------
+  # Linux ARM Wheels
+  # ---------------------------------------------------------------------------
+
+  linux-arm:
```

Comment:
So, the reason why I broke this out into a separate job is cuz I want to make it as straightforward as possible to switch the aarch64 builds to a linux ARM runner as soon as they become available -- in a few months, I'm guessing (ARM runners were recently made widely available to enterprise accounts; maybe ASWF has access?). These emulated builds take an order of magnitude longer than the native builds.

The other main difference between the linux x86_64 and aarch64 jobs is that I'm skipping the tests for the aarch64 jobs. It seems CIBW sometimes has trouble running tests on cross-compiled wheels. I've verified locally that the aarch64 build artifacts behave properly.

(likewise, the macos-arm64 tests failed until I moved the builds to the macos-14 (ARM) runner)

Beyond that, I do have some thoughts about other ways we could run these builds -- I think I'd like to switch to incremental builds governed by the CIBW config rather than discrete github jobs -- but for this initial PR, I thought I'd try to match as closely as possible to what OCIO is doing (simply cuz I'm familiar with OCIO, and things seem to be working pretty well over there).

Reply:
I don't remember the exact reasons why we split the jobs like this, I like the simplicity it gives when reviewing workflows and seeing what works or doesn't at a single glance. One other benefits is that it runs in parallel which is important as OCIO is a bit long to compile. But maybe cibuildwheel can do that too from a single job?

OCIO setup is working but might be a bit old at this point and no longer following latest recommendations. I think we should move to using scikit-build for example.

Amazing work @zachlewis by the way! Will try to have a look this week.

Reply:
> But maybe cibuildwheel can do that too from a single job?

Sort of. The incremental builds run in serial, but they can share a build cache. If you use a persistent build directory, I think it'll reuse the static dependency libs from the first build for all the subsequent Python module builds (although I haven't tried it myself). It is particularly helpful here, because the OIIO build system is also building OCIO, OpenEXR, TIFF, and handful of others (~12 min on the current runners), all of which can just be built once and reused for all the module builds. It doesn't take too much longer (unless you're doing emulated builds), and it seems like it would save a lot of energy...

Reply:
Interesting. Assuming you have some limitations like you can't re-use the same dependencies build between manylinux 2014 and manylinux 2.18 for example?

Reply:
Yeah, you do -- I guess "persistent build directory" is kind of a loaded term. I'm setting `$SKBUILD_BUILD_DIR=/tmp/build_oiio`, which, for linux, is container-specific storage (as opposed to a volume mapped to host storage), so it kind of works itself out neatly. On MacOS, things get a bit hairier, as far as universal2 / multi-arch builds are concerned. But as long as I'm using a macos-14 runner for the ARM64 wheels, and macos-13 or macos-12 runners for the x64 wheels, there doesn't seem to be a problem.

Reply:
>So, the reason why I broke this out into a separate job is cuz I want to make it as straightforward as possible to switch the aarch64 builds to a linux ARM runner as soon as they become available -- in a few months, I'm guessing (ARM runners were recently made widely available to enterprise accounts; maybe ASWF has access?). These emulated builds take an order of magnitude longer than the native builds.

You already set `arch` as a variant, so you should be able to do

```yaml
runs-on: ${{ matrix.runs-on }}
strategy:
  matrix:
    include:
    - build: CPython 3.8 ARM 64 bits manylinux2014
      manylinux: manylinux2014
      python: cp38-manylinux_aarch64
      arch: aarch64
      runs-on: <image tag>
```

Anyway, I think the way it is is fine. It's just duplicate configs, but if Larry is fine with that, then that's what's important.

Reply:
oooh, I see what you're saying. That's a pretty good idea.